gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -309,7 +311,7 @@ local function CreateEquipmentList(t)
 			local ic = nil
 
 			-- Create icon panel
-			if item.material and item.material ~= "vgui/ttt/icon_id" then
+			if item.ttt2material then
```

Comment:
I'd suggest, to use a more fitting name, that expresses the function of the variable more accurately. It should show, that this is used for caching.